utils/paymentRedirect.ts

```diff
@@ -16,8 +16,8 @@ export const paymentRedirect = (ticketInit: TicketInitDto, paymentOption: Paymen
       paymentOption === 'apple-pay'
         ? ticketInit.paymentUrls.paymentUrlAPAY
         : paymentOption === 'google-pay'
-        ? ticketInit.paymentUrls.paymentUrlGPAY
-        : ticketInit.paymentUrls.paymentUrlCard
+          ? ticketInit.paymentUrls.paymentUrlGPAY
+          : ticketInit.paymentUrls.paymentUrlCard
```

Comment:
Just checking formatting, I think this is not forced by prettier, but by you IDE and it should not be. (Not a blocker for merging)

Reply:
this was an error in ts:check